Selectors/src/main/java/io/doist/recyclerviewext/choice_modes/Selector.java

```diff
@@ -24,7 +24,7 @@
     protected final RecyclerView mRecyclerView;
     protected final RecyclerView.Adapter mAdapter;
 
-    protected OnSelectionChangedListener mObserver;
+    private List<OnSelectionChangedListener> mObservers;
```

Comment:
Why do we allow for this list to be `null`? We could initialize it here, i.e. `private List<OnSelectionChangedListener> mObservers = ArrayList<>()` and then, in the methods below, just add/remove listeners as requested instead of nullifying the list. 🤔

Reply:
Let's say it was inspired by some Android classes 😬